File: 백준/Bronze/10101. 삼각형 외우기/삼각형 외우기.js
const [a, b, c] = require('fs').readFileSync(0, 'utf-8').toString().trim().split('\n').map(Number);

if (a + b + c !== 180) {
    console.log('Error');
} else if (a === b && b === c && c === a) {
    console.log('Equilateral');
} else if (a === b || b === c || c === a) {
    console.log('Isosceles');
} else {
    console.log('Scalene');
}

File: 백준/Bronze/10101. 삼각형 외우기/삼각형 외우기.py
import sys

A, B, C = map(int, sys.stdin.readlines())

if A + B + C != 180:
    print("Error")
elif A == B and B == C and C == A:
    print("Equilateral")
elif A != B and B != C and C != A:
    print("Scalene")
else:
    print("Isosceles")